front/src/modules/ui/data-table/table-cell/components/TableCellContainer.tsx

```diff
@@ -64,7 +64,8 @@ export const TableCellContainer = ({
   const hasSoftFocus = useIsSoftFocusOnCurrentTableCell();
 
   const setSoftFocusOnCurrentTableCell = useSetSoftFocusOnCurrentTableCell();
-  const { isSomeCellInEditMode } = useIsSomeCellInEditMode();
+  const { useIsSomeCellInEditMode } = useSomeCellInEditMode();
+  const cellInEditMode = useIsSomeCellInEditMode();
```

Comment:
It's not a hook that is returned by useRecoilCallback but a function that should be named like `getIsSomeCellInEditMode` and it should be called in your handler directly.